Autopilot/Src/altimeter.c

```diff
@@ -1,61 +1,48 @@
 #include "altimeter.h"
 
-#define MPL3115A2_ADDR 0x60 << 1
-#define MPL3115A2_I2C I2C2
+#define MS5637_ADDR 0x76 //Address for the MS5637. Needs to be shifted for write/read (ref:datasheet)
```

Comment:
you can alternatively have MS5637_READ 0xED MS5637_WRITE 0xEC defined so you don't have to shift

Reply:
Ya that would actually be better, ty